vms/vm-under-test/scripts/first-boot

```diff
@@ -19,8 +19,9 @@
 
 set -e
 
-systemctl mask "$(systemctl --type swap | grep '.swap' | awk '{print $1}')"
-
+if systemctl --type swap list-units | grep -q '.swap'; then
+  systemctl mask "$(systemctl --type swap list-units | grep '.swap' | awk '{print $1}')"
+fi
```

Comment:
Could you please give a few words in the commit message about what these commands do?

Reply:
DONE